article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -170,63 +142,28 @@ object ArticlePicker {
         "isNotPaidContent",
       ),
     )
-    article100PercentPageFeatures.forall({ case (test, isMet) => isMet })
-  }
-
-  def dcrDisabled(request: RequestHeader): Boolean = {
-    val dcrEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
```

Comment:
can we get rid of this switch now too? Have I missed that somewhere?

Reply:
It's maybe still useful as an absolute last ditch fallback in case something goes badly wrong with the DCR server farm? At some point though it will become a liability because turning it off will cause more issues than it might fix (as we start to add new features to DCR / delete Frontend code)